AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -302,6 +302,7 @@ texture-packs.com#%#//scriptlet("abort-on-property-read", "mdpDeBlockerDestroyer
 !
 !
 ||smedata.sk/monitor/bundles/smeblockmanagerapi/build/js/gfcc.js
+@@||snowheroes.io/ads.js
```

Comment:
What about content blockers? If they require this exception, it won't work here.
In this case, you need to duplicate it to content blockers section. Please check.

Reply:
Why this rule will not work here for content blockers?

Reply:
Because blocking rule exists in Base filter (general) and you add an exception to Annoyances filter (privacy). They are not related, so, you need to add to content blockers section (general) or use `!#safari_cb_affinity(general)` directive.

Reply:
According to the latest information, it is better to use the directive

Reply:
ok, thanks